charts/minibroker/templates/service.yaml

```
@@ -10,4 +10,4 @@ spec:
   ports:
   - protocol: TCP
     port: 80
-    targetPort: 8080
+    targetPort: {{ .Values.broker.deployment.port }}
```

```suggestion
    targetPort: broker
```
I find pointing at the port _name_ to be generally clearer.

---

Fixed in https://github.com/kubernetes-sigs/minibroker/pull/81/commits/e2dbcfd44fc89d859186db6608d199a3d28f67ec.